src/components/Products.js

```diff
@@ -0,0 +1,47 @@
+// import React, { Component } from 'react'
+
+//  export default class Items extends Component {
+//     render() {
+//         return (
+//             <div>
+
+//                 <h3>Item Name</h3>
+//                 <p>Price</p>
+//                 <button>Add to cart</button>
+//                 <p>Picture</p>
+//                 <p>Item Description</p>
+
+//             </div>
+//         )
+//     }
+// }
+
+import React from "react";
+
+const Products = (props) => {
+  //   console.log(props);
+  //   console.log(props.keyOfProducts[0].name);
+    let product = props.keyOfProducts[0];
+  //   console.log(product);
+
+  let allProducts = props.keyOfProducts.map((eachProduct, e) => {
+    return (
+      <li key={props.id}>
+        <h4>{eachProduct.name}</h4>
+        <img src={eachProduct.img} alt={eachProduct.name} />
+        <br></br>
+        {eachProduct.description}
+        <br></br>${eachProduct.price} <button>Add to Cart</button>
+      </li>
+    );
+  });
```

Comment:
This looks great! This is exactly the way you want to do it. Alternatively you could just put the whole .map function inside the return, then you don't have to create a new variable. But that's just a preference thing.